.github/workflows/ci.yml

```
@@ -63,9 +63,7 @@ jobs:
         with:
           python-version: '3.10'
           cache: 'pip'
-      - name: Install package
-        run: |
-          - run: pip install -r docs/requirements.txt -e .
+      - run: pip install -r docs/requirements.txt -e .
```

```suggestion
      - run: pip install -e ".[docs]"
```

This is equivalent and simpler. 😄

---

I was doing it like the documentations on Read the docs, to test first.